pep-0394.txt

```diff
@@ -175,14 +265,17 @@ making such a change.
   versa. That way, if a sysadmin does decide to replace the installed
   ``python`` file, they can do so without inadvertently deleting the
   previously installed binary.
-* If the Python 2 interpreter becomes uncommon, scripts should nevertheless
-  continue to use the ``python3`` convention rather that just ``python``. This
-  will ease transition in the event that yet another major version of Python
-  is released.
+* Even as the Python 2 interpreter becomes less common, it remains reasonable
+  for scripts to continue to use the ``python3`` convention, rather than just
+  ``python``. In the event of a future Python 4.0 release, distributors may
```

Comment:
I'd suggest leaving out any mention of Python 4.  1) it can create FUD; 2) it's entirely vaporware; 3) we don't even know whether there *will* be a Python 4 or what it will look like.  We can always add a Python 4 recommendation when the time comes.

Reply:
Agreed. Good catch @warsaw